apps/website/src/components/wallet/video-section.tsx

```diff
@@ -0,0 +1,76 @@
+import { ContextTag, Text } from '@status-im/components'
+
+import { ParalaxCircle } from './parallax-circle'
+
+const VideoSection = () => {
+  return (
+    <div className="relative flex flex-col md:flex-row">
+      <ParalaxCircle initialLeft={-100} initialTop={-100} />
+
+      <div className="relative z-[1] flex flex-col px-5 pt-24 lg:px-[164px] lg:pt-[240px]">
+        <h1 className="text-[40px] font-bold leading-[44px] lg:text-[64px] lg:leading-[68px]">
+          Fully
+          <br />
+          Decentralized
+          <br />
+          Networks
+        </h1>
+
+        <div className="flex max-w-[462px] flex-col pt-4">
+          <Text size={27}>
+            Status supports blockchain networks that are fully committed to
+            decentralization.
+          </Text>
+          <div className="pt-8">
+            <Text size={13} color="$neutral-50" weight="medium">
+              Currently supported networks
+            </Text>
+          </div>
+          <div className="flex pt-3">
+            <div className="pr-[10px]">
+              <ContextTag
+                type="network"
+                network={{
+                  name: 'Mainnet',
+                  src: '/images/wallet/ethereum.png',
+                }}
+                size={24}
+              />
+            </div>
+            <div className="pr-[10px]">
+              <ContextTag
+                type="network"
+                network={{
+                  name: 'Optmism',
+                  src: '/images/wallet/optimism.png',
+                }}
+                size={24}
+              />
+            </div>
+            <div>
+              <ContextTag
+                type="network"
+                network={{
+                  name: 'Arbitrum',
+                  src: '/images/wallet/arbitrum.png',
+                }}
+                size={24}
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className="relative right-[-5px] top-0 flex justify-center md:absolute">
+        <video autoPlay loop muted playsInline>
+          <source
+            src="/images/wallet/vitalik.mp4"
+            type="video/mp4;codecs=hvc1"
+          />
+          <source src="/images/wallet/vitalik.webm" type="video/webm" />
```

Comment:
I like that you want to keep images and videos for a feature page under one folder. However, I don't like putting videos under `/images` folder. I would an `assets` folder where we can group it based on page. What do you think?